astropy/modeling/functional_models.py

```diff
@@ -91,6 +91,32 @@ class Gaussian1D(Fittable1DModel):
     amplitude = Parameter(default=1)
     mean = Parameter(default=0)
     stddev = Parameter(default=1)
+    _bbox = 'auto'
+
+    @property
+    def bbox(self):
```

Comment:
So this is starting to sound exactly like the `bounds` property. Do we really need a new `bbox` property?